src/lib.rs

```diff
@@ -88,20 +90,29 @@ impl Contract {
         }
     }
 
-    pub fn add_like(&mut self, post_id: PostId) {
+    #[payable]
+    pub fn add_like(&mut self, post_id: PostId) -> Promise {
         near_sdk::log!("add_like");
         let mut post: Post = self
             .posts
             .get(post_id)
             .unwrap_or_else(|| panic!("Post id {} not found", post_id))
             .into();
+        let post_author = post.author_id.clone();
         let like =
             Like { author_id: env::predecessor_account_id(), timestamp: env::block_timestamp() };
         post.likes.insert(like);
         self.posts.replace(post_id, &post.into());
+        notify::notify_like(post_id, post_author)
```

Comment:
Same thing, I don't think we need to wait on the other contract to execute, so we can probably drop the promise without returning it.

Reply:
OK!